### PR TITLE
Release preparation

### PR DESCRIPTION
**Pull request by kvnify**
I've done a bit of a tidy. Fixed a failing test and updated the pod spec.  Let me know if there's anything else that needs too e done.  I'd like to merge soon and then update master shortly thereafter and get cocoa pod updated. 

I also updated the pods and project settings to work with the latest Xcode.

@robb Do you think it's worth adding Carthage support?

### CODE REVIEW

File: UnderscoreTests/Strings/UnderscoreStringsSpec.m [hunk 1]
```diff
@@ -198,17 +207,32 @@
...
+        expect(Underscore.split(@"It was the best of times, it was the worst of times"
+                                , @"t")).to.equal((@[ @"I",
+                                                      @" was ",
+                                                      @"he bes",
+                                                      @" of ",
+                                                      @"imes, i",
+                                                      @" was ",
+                                                      @"he wors",
+                                                      @" of ",
+                                                      @"imes"]));
```
Comment (robb) [inline UnderscoreTests/Strings/UnderscoreStringsSpec.m]: This is still kinda hard to parse, maybe we should just use a shorter string to split?

**Comment by robb** [comment]
This looks great! Just one note :sparkling_heart:

**Comment by robb** [comment]
What about setting up travis-ci for this?

**Comment by kvnify** [comment]
I'll look into it.

File: UnderscoreTests/Strings/UnderscoreStringsSpec.m (hunk 1 above)
Comment (kvnify) [inline UnderscoreTests/Strings/UnderscoreStringsSpec.m]: Sure thing.

**Comment by kvnify** [comment]
Just want to double check that you are good with me merging this now? I'd love to try and have the updated pod rolled out this week and start doing some of the legwork for 0.4.0

**Comment by robb** [comment]
Would you mind setting up CI on this first? Cool if not ✌

**Comment by kvnify** [comment]
Ok. I'll see if I can get that going tonight.

**Comment by robb** [comment]
Thanks 🙇

**Comment by robb** [comment]
> Do you think it's worth adding Carthage support?

Never got back to you on that one. I think it's desirable but not critically urgent. :v:

**Comment by robb** [comment]
:cherry_blossom:
